Gruntfile.coffee

```diff
@@ -103,6 +103,10 @@ module.exports = (grunt) ->
       main:
         files: [
           {
+            src: ['api-proxy-sign']
```

Comment:
This file contains the signature for using the generic web API proxy. See https://github.com/salsita/web-api-proxy for more details.